_overviews/scala3-book/types-type-classes.md

```diff
@@ -23,7 +23,7 @@ In Scala 3, _type classes_ are just _traits_ with one or more type parameters, l
 trait Show[A]:
   def show(a: A): String
 ```
-Instances of `Show` for a particular type `A` witness that `A` we can show an instance of type `A`.
+Instances of `Show` for a particular type `A` witness that we can show (ie, produce a text representation of) an instance of type `A`.
```

Comment:
Thanks, this is a good addition. I would just change “ie,” to “i.e.,”.

If you’re interested in another change in this file, I just saw a heading with this text:

#### Automatically passing Type Class Instances

I think that should be:

#### Automatically passing type class instances

I can change this separately if you prefer, but I just noticed it and thought I could/should mention it now.

Reply:
Thanks, I’ve applied both suggestions.